api/src/test/java/org/pmiops/workbench/api/config/TestBigQueryConfig.java

```diff
@@ -0,0 +1,56 @@
+package org.pmiops.workbench.api.config;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.gson.Gson;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.liquibase.LiquibaseAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.UUID;
+
+@SpringBootApplication(exclude = LiquibaseAutoConfiguration.class)
+public class TestBigQueryConfig {
+
+    @Bean
+    public BigQuery bigQueryService() throws Exception {
+        InputStream keyStream = TestBigQueryConfig.class.getClassLoader().getResourceAsStream(
+            "bq-test-key.json");
+        if (keyStream == null) {
+            keyStream = new FileInputStream(new File("src/test/resources/bq-test-key.json"));
+            if (keyStream == null ) {
```

Comment:
Remove this check; FileInputStream will throw FileNotFoundException if the file doesn't exist.

You may want to add a comment saying we're doing this to support Intellij.

Reply:
Removed.